commit f2022e428f594c3370878b44cd0eaa2d9fdbe39f (refs/changes/65/565/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-30T15:15:39-07:00 (3 years ago)
    
    joyent/node-triton#136 bash completion for 'triton profile create --copy <TAB>'

diff --git a/CHANGES.md b/CHANGES.md
index 56543bf..a9435d6 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -7,6 +7,7 @@ Known issues:
 
 ## not yet released
 
+- [#136] bash completion for `triton profile create --copy <TAB>`
 
 ## 4.14.0
 
diff --git a/lib/do_profile/do_create.js b/lib/do_profile/do_create.js
index 6a9f979..f5dcd60 100644
--- a/lib/do_profile/do_create.js
+++ b/lib/do_profile/do_create.js
@@ -352,8 +352,9 @@ do_create.options = [
     {
         names: ['copy'],
         type: 'string',
-        helpArg: 'NAME',
-        help: 'A profile from which to copy values.'
+        helpArg: 'PROFILE',
+        help: 'A profile from which to copy values.',
+        completionType: 'tritonprofile'
     },
     {
         names: ['no-docker'],
